### DISCUSSION TITLE
google analytics implementation

### DISCUSSION THREAD

**Discussion by AdesonBezerra** [Sketchware-Pro/Sketchware-Pro#461] [Ideas]
google analytics has great analytics and should not be difficult to implement in sketchware pro 

I tried to implement normally by downloading the libs but the sketchware does not compile correctly and gives an error. 
![Screenshot_20220602-131139_Coclew](https://user-images.githubusercontent.com/106257193/172012844-87d9c5c6-9aea-4ded-9708-c4eb9cbaeb08.png)
![Screenshot_20220602-131534_Sketchware Pro](https://user-images.githubusercontent.com/106257193/172012851-5bc64635-51b2-4c16-9529-0285b51db36f.png)
![Screenshot_20220602-131947_MT Manager](https://user-images.githubusercontent.com/106257193/172012856-f5887a60-c387-4a8c-afad-b3437fee373d.png)

**Comment by H-uey** [comment]
Do you have any updates, was it successful?
